__tests__/intergration.test.js

```diff
@@ -109,14 +109,38 @@ describe("GET /api/articles",()=>{
         })
     })
     //QUERIES=========================================
-    xtest("GET: 200 - Retrieved all data from articles sorted by the column queried",()=>{
+    test("GET: 200 - Retrieved all data from articles sorted by the column queried",()=>{
         return request(app)
         .get("/api/articles?sort_by=author")
         .expect(200)
         .then(({body: {articles}})=>{
             expect(articles).toBeSortedBy("author",{descending: true})
         })
     })
+    test("GET: 200 - Retrieved all data from articles sorted by the column queried AND can be changed to order by ascending or descending",()=>{
+        return request(app)
+        .get("/api/articles?sort_by=topic&order=ASC")
+        .expect(200)
+        .then(({body: {articles}})=>{
+            expect(articles).toBeSortedBy("topic",{descending: false})
+        })
+    })
+    test("GET: 200 - Retrieved all data from articles sorted ascending or descending",()=>{
+        return request(app)
+        .get("/api/articles?order=ASC")
+        .expect(200)
+        .then(({body: {articles}})=>{
+            expect(articles).toBeSortedBy("created_at",{descending: false})
+        })
+    })
+    test("Error: 404 - Invalid Query Given",()=>{
+        return request(app)
```

Comment:
This is good but we should really be testing for an invalid order and an invalid sort_by separately. Similar tests, but slightly different functionality.